apps/editor.planx.uk/src/@planx/components/Pay/Public/InviteToPayForm.tsx

```diff
@@ -86,59 +83,14 @@ const InviteToPayForm: React.FC<InviteToPayFormProps> = ({
   yourDetailsLabel,
   paymentStatus,
 }) => {
-  const [sessionId, path] = useStore((state) => [state.sessionId, state.path]);
-  const [isLoading, setIsLoading] = useState(false);
-  const [error, setError] = useState(false);
```

Comment:
No boilerplate required for the request lifecycle - we get this as `isPending` and `error` from the `useMutate()` hook.